splice_machine/src/test/java/com/splicemachine/db/impl/sql/compile/MergeJoinSelectivityIT.java

```diff
@@ -51,6 +51,16 @@ public void innerJoin() throws Exception {
                     "explain select * from --splice-properties joinOrder=fixed\n ts_10_spk, ts_5_spk --splice-properties joinStrategy=MERGE\n where ts_10_spk.c1 = ts_5_spk.c1",
                     "rows=10","MergeJoin");
         }
+
+        try(Statement s = methodWatcher.getOrCreateConnection().createStatement()){
+            rowContainsQuery(s,
+                    new int[]{1,3},
+                    "explain select upper(ts_10_spk.c2), upper(ts_5_spk.c2) from --splice-properties joinOrder=fixed\n " +
+                            "ts_10_spk --splice-properties index=ts_10_spk_expr_idx\n, " +
+                            "ts_5_spk --splice-properties index=ts_5_spk_expr_idx, joinStrategy=MERGE\n " +
+                            "where upper(ts_10_spk.c2) = upper(ts_5_spk.c2)",
+                    "rows=10","MergeJoin");
+        }
```

Comment:
afaik you can just do SpliceUnitTest.rowContainsQuery(levels, query, "rows=10","MergeJoin"); which does the try/createStatement for you

Reply:
That's right. Currently, `xJoinSelectivityIT` tests are derived from `BaseJoinSelectivityIT` but not from `SpliceUnitTest`. So we don't have the `rowContainsQuery()` here. The one we see in these tests are actually a method in `BaseJoinSelectivityIT` that does the same...

All these tests probably should be adapted to derive from `SpliceUnitTest`. But I guess I'll do this in the very end since they are just tests.